### ISSUE TITLE
Komodo 9 tries to update to K10 only addons

### ISSUE THREAD

**Issue by babobski** [Komodo/KomodoEdit#1578]
### Short Summary
Noticed a changed some of my addons to min version to Komodo 10.
Only when i open Komodo 9 I get update warnings on these addons.
When i try to update them, I get the message that the addon is not compatible with the current Komodo version.

### Platform Information
Komodo IDE 9. 

### Additional Information
I removed the autocompletion function out of my less compiler, because this is now native supported by Komodo. That's why I Bumped the less compiler to K10 only but in Komodo 9 I get this update warning:
![2016-05-22 09_55_25](https://cloud.githubusercontent.com/assets/3530262/15452994/59dda69e-2003-11e6-922b-7fc08a7852f6.png)
Also I noticed the version numbers are wrong, It should read 2.1.0 > 2.0.5 instead of 2.0.5. > 2.1.0.

**Comment by mitchell-as** [comment]
I'm a bit confused. You are running Komodo 9 with an add-on, but you've since changed that add-on to require Komodo 10 minimum. Then Komodo 9 tells you it needs to update the add-on, but when it tries to, it says the add-on isn't compatible with Komodo 9?

**Comment by babobski** [comment]
Yes, i have updated my less compiler to komodo [10 only](https://github.com/babobski/Komodo-LESS-Compiler/blob/master/install.rdf#L21) and my and the EOL widget. because of all the changes i didn't want to support both versions.
Only in komodo 9 I get a update warning for these changes only when i try to update them they fail because they are not compatible.

To reproduce:
Install the last [komodo 9 version](https://github.com/babobski/EOL-widget/releases/download/2.0/EOL-widget-2.0.xpi) of the EOL widget.
It will ask you to reboot, now on start up you get the "package update available" message
![2016-05-23 22_25_01](https://cloud.githubusercontent.com/assets/3530262/15483512/61f94ff8-2135-11e6-91fa-f749978064e7.png)
If you click on it you will get the update screen:
![2016-05-23 22_28_23](https://cloud.githubusercontent.com/assets/3530262/15483603/c8199ee6-2135-11e6-97ea-6c7197f9688e.png)
If you now try to update the addon it fails because the version is not compatible:
![2016-05-23 22_30_04](https://cloud.githubusercontent.com/assets/3530262/15483635/f35cec7a-2135-11e6-9513-d54b28707ba7.png)

**Comment by mitchell-as** [comment]
Okay thanks. It sounds like Komodo's package manager doesn't check to see if an update is actually compatible with the current version of Komodo before notifying that an update is indeed available. Thanks for the extra info.

**Comment by Naatan** [comment]
I'll change this server side so that Komodo 9 users don't even see addons that have a minVersion over 9.
